pkgs/by-name/ca/calaos_installer/package.nix

```
@@ -1,31 +1,29 @@
 {
-  mkDerivation,
   lib,
   stdenv,
   fetchFromGitHub,
-  qmake,
-  qttools,
-  qtbase,
+  libsForQt5,
 }:
 
-mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "calaos_installer";
   version = "3.11";
 
   src = fetchFromGitHub {
     owner = "calaos";
     repo = "calaos_installer";
-    rev = "v${version}";
     sha256 = "sha256-e/f58VtGmKukdv4rIrGljXhA9d/xUycM5V6I1FT5qeY=";
+    rev = "v${finalAttrs.version}";
```

```suggestion
    hash = "sha256-e/f58VtGmKukdv4rIrGljXhA9d/xUycM5V6I1FT5qeY=";
    tag = "v${finalAttrs.version}";
```
if we are modernizing